android/buildAndroidBOINC-CI.sh

```diff
@@ -119,6 +119,7 @@ if [ "x$build_dir" != "x" ]; then
 else
     cd ../
     BUILD_DIR="$(pwd)/3rdParty/android"
+    TRD_PARTY="$(pwd)/3rdParty"
```

Comment:
It's not obvious that `TRD_PARTY` means third-party. Please rename to `THIRD_PARTY`

Reply:
ok.

Reply:
done.